src/corehost/cli/exe/apphost/CMakeLists.txt

```diff
@@ -3,6 +3,12 @@
 
 cmake_minimum_required (VERSION 2.6)
 set(DOTNET_HOST_EXE_NAME "apphost")
+
+if (NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
```

Comment:
So we wont be able to carry libraries locally for OSX? It will be good to add a comment about this here to provide context.

LGTM otherwise.